src/2/send-msg/native-polls-msg.md

```diff
@@ -0,0 +1,59 @@
+# Native Poll Messages
+
+[![native poll tests](https://img.shields.io/badge/Examples-Native_Polls-green.svg?style=flat-square)](https://github.com/TelegramBots/Telegram.Bot/blob/master/test/Telegram.Bot.Tests.Integ/Polls/PollMessageTests.cs)
+
+Bots just as regular users can send native polls only to groups and channels, but not to private chats.
+
+## Send a poll
+
+[![sendPoll method](https://img.shields.io/badge/Bot_API_method-send_poll-blue.svg?style=flat-square)](https://core.telegram.org/bots/api#sendpoll)
+
+This is the code to send a poll to a chat.
+
+```c#
+ChatId chatId = "@group_or_channel_username";
+Message pollMessage = await botClient.SendPollAsync(
+    chatId: chatId,
+    question: "Did you ever hear the tragedy of Darth Plagueis The Wise?",
+    options: new []
+    {
+        "Yes for the hundredth time!",
+        "No, who`s that?"
+    },
+    replyMarkup: ... // optional IReplyMarkup
```

Comment:
we try to provide copy-pastable examples (as close as possible) for user in the doc examples.
if you have a suggestion for reply markup, include the working example. the argument could also be commented out similar to the audio message examples.
```suggestion
```